backend/doc/setting.md

```diff
@@ -1,184 +1,40 @@
-# Setting up Tracim #
-
-## Parameters Names
-
-Most settings in Tracim are configurable using both the INI configuration file and environment variables.
-
-### Tracim fully supported variables:
-
-You can set those parameters in INI configuration file (see `config_file_name`) or
-environnement variable (see `env_var_name`).
-
-The priority order is (from less to more priority):
-- default values
-- configuration file
-- environnement variables
-
-<!--- Maintainer: use tracimcli dev parameters list --template "| {env_var_name: <74}| {config_file_name: <63}| {config_name: <67}|"  with all apps enabled to update this list properly --->
```

Comment:
You have removed the explaination to generate the list ?

Reply:
This is not the first time someone tries to remove this line. I already did once.

It looks like a leftover. Maybe we should just put it as regular text÷

Reply:
I think also its good for all user to know what we need to do to generate this list.

Reply:
Sorry, I forgot to re-introduce it in `env_settings.md`, I'll do that.